packages/engine/Source/Scene/Model/ModelAnimationChannel.js

```diff
@@ -240,10 +241,10 @@ function initialize(runtimeChannel) {
   switch (path) {
     case AnimatedPropertyType.TRANSLATION:
     case AnimatedPropertyType.SCALE:
-      scratchVariable = new Cartesian3();
+      scratchCartesian3 = new Cartesian3();
```

Comment:
@javagl Correct me if I'm wrong, but is this switch statement needed anymore? We can just initialize the scratch variables at the module scope, as we do for most cases.

```
const scratchCartesian3 = new Cartesian3();
const scratchQuaternion = new Quaternion();
```

Reply:
Sorry, I misinterpreted the comment at https://github.com/CesiumGS/cesium/issues/11685#issuecomment-1852214923 

When it's OK that both are initialized (even if one of them may remain unused), then the switch statement can indeed be removed.